protocol/activity_center_persistence.go

```diff
@@ -61,7 +61,10 @@ func (db sqlitePersistence) DeleteActivityCenterNotificationForMessage(chatID st
 
 	for _, notification := range notifications {
 		if notification.LastMessage != nil && notification.LastMessage.ID == messageID {
-			withNotification(notification)
+			switch notification.Type {
+			case ActivityCenterNotificationTypeNewOneToOne, ActivityCenterNotificationTypeNewPrivateGroupChat:
+				withNotification(notification)
+			}
```

Comment:
Here we're focusing on only removing notifications that might have a custom `LastMessage` from creating a `NewOneToOne` or `NewPrivateGroupChat` notification. Although it's likely that we won't need this anymore since we no longer use the `LastMessage` field when creating those notifications.

If it makes sense to delete this `if-block` then that's what I'll do next.

Reply:
Looks to me like `LastMessage` can be completely removed 👍